src/google.cr

```diff
@@ -661,6 +661,8 @@ class Google::Calendar::Event
         organizer: attendee.organizer)
     end
 
+    hide_attendees = @guests_can_see_other_guests.nil? ? true : @guests_can_see_other_guests
```

Comment:
Shouldn't this be inversed, i.e.
`guests_can_see_other_guests = true` then this should be `hide_attendees = false`
I assume the default is `true` so if it's `nil` then `hide_attendees = false`

so the code should be:
```suggestion
    hide_attendees = @guests_can_see_other_guests.nil? ? false : !@guests_can_see_other_guests
```